mw-webapp/src/component/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,36 @@
+import {ReactElement, useState} from "react";
+import {Root as DialogRoot} from "@radix-ui/react-dialog";
```

Comment:
@radix-ui/react-dialog does not added to dependencies in the package.json and it leads to an error even if I run "pnpm i" command

Reply:
My bad, fixed